clang/lib/DPCT/CUBAPIMigration.cpp

```diff
@@ -1139,6 +1141,17 @@ void CubRule::processBlockLevelMemberCall(const CXXMemberCallExpr *BlockMC) {
     CubParamAs << GroupOrWorkitem << InEA.getReplacedString() << OpRepl;
     Repl = NewFuncName + "(" + ParamList + ")";
     emplaceTransformation(new ReplaceStmt(BlockMC, Repl));
+  } else if (FuncName == "Load") {
+  
+    GroupOrWorkitem = DpctGlobalInfo::getItem(BlockMC);
+    NewFuncName = Mapnames::getClNamespace() + "load";
```

Comment:
The mapping seems not correct, I try to compile lit int this PR, but it failed.

Reply:
I added in the Load mapping in the expression (cxxmembercallexpr).